extensions/sherlock-rxjs/rxjs.test.ts

```diff
@@ -252,5 +273,37 @@ describe('rxjs/rxjs', () => {
                 expect(e.message).toBe('my error');
             }
         });
+
+        it('should reconnect after a disconnect, allowing to continue after a completed Observable', () => {
+            let connections = 0;
+            let subj: Subject<string> = 0 as any;
+            const obs = defer(() => (++connections, subj = new Subject<string>()));
+            const when = atom(true);
+            const d$ = fromObservable(obs);
```

Comment:
I would have expected `finalizing` a `Derivable` when the `Observable` completes.
Since `Observable`s aren't inherently shared, one can complete and the next subscriber won't mind. With `Derivable`s this is different. You may get a wildly different result if you are alone, from when you aren't. And you may not be able to close all reactors. In that case it may be nice to be explicit and make the `Derivable` final, so you can create a new one if needed.

Reply:
You are absolutely right! 🤦‍♂ again...